OmsAgent/omsagent.py

```diff
@@ -200,7 +205,28 @@ def main():
     # Finish up and log messages
     log_and_exit(operation, exit_code, message)   
         
+def check_disk_space_availability():
+    """
+    Check if there is the required space on the machine.
+    """
+    try:
+        if get_free_space_mb("/var") < 200 or get_free_space_mb("/etc") < 200:
+            # 52 is the exit code for missing dependency i.e. disk space
+            # https://github.com/Azure/azure-marketplace/wiki/Extension-Build-Notes-Best-Practices#error-codes-and-messages-output-to-stderr
+            return 52
+        else:
+            return 0
+    except:
+        print('Failed to check disk usage.')
+        return 0
+        
 
+def get_free_space_mb(dirname):
+    """
+    Get the free space in MB in the directory path.
+    """
+    st = os.statvfs(dirname)
```

Comment:
can we check for failure/-1 here?  shouldn't happen, but in case there is some crazy distro we should handle error and proceed?